grails-app/views/wps/jobComplete.gsp

```diff
@@ -1,13 +1,13 @@
 Hi,
 
-Your IMOS subset has been created. You can download it by clicking here: ${jobReportUrl}
+Your ${grailsApplication.config.portal.siteAcronym} subset has been created. You can download it by clicking here: ${jobReportUrl}
```

Comment:
@pmbohm - refer https://github.com/aodn/aodn-portal/pull/2220 - grailsApplication is not available in this template and the above results in a null pointer exception.